app/controllers/patients_controller.rb

```diff
@@ -98,9 +98,6 @@ def create
     # Set the creator as the current user
     patient.creator = current_user
 
-    # Set the subject jurisdiction to the creator's jurisdiction
-    patient.jurisdiction = current_user.jurisdiction
```

Comment:
Just in case, I'd like to see this set by default if something ever goes wrong with the selection

Reply:
Oh, I guess this is where I could do the server side validation for patient.jurisdiction_id? I think it would always have some kind of default value since this has been added: https://github.com/SaraAlert/SaraAlert/pull/63/files#diff-4f4c29154577c3b2ecf2075ade80b28eR31
but I do think it would be a good idea to just verify that the selected jurisdiction is actually valid and under the hierarchy of the current user, so I'll put that in.